samples/net/updatehub/prj.conf

```diff
@@ -0,0 +1,25 @@
+# Turn on the updatehub
+CONFIG_UPDATEHUB=y
+
+#Example of product_uid, you must set you product uid.
+CONFIG_UPDATEHUB_PRODUCT_UID="e4d37cfe6ec48a2d069cc0bbb8b078677e9a0d8df3a027c4d8ea131130c4265f"
+
+#example for the updatehub will poll on the server request for updates (min: 1 minute, máx: 30 days)
+CONFIG_UPDATEHUB_POLL_INTERVAL=1
+
+CONFIG_UPDATEHUB_CE_SERVER=y
+CONFIG_UPDATEHUB_SERVER="<your-addres-ip>"
+
+#Minimal Heap mem pool size for the updatehub working
+CONFIG_HEAP_MEM_POOL_SIZE=2048
+CONFIG_SYSTEM_WORKQUEUE_STACK_SIZE=2048
+
+CONFIG_NET_DHCPV4=y
+
+
+#Optional if you would like test on the your server
+CONFIG_UPDATEHUB_SHELL=y
+
+# Debug helpers
+CONFIG_UPDATEHUB_LOG=y
+CONFIG_UPDATEHUB_LOG_LEVEL=3
```

Comment:
You need to use the log level template file, after that this setting would be called CONFIG_UPDATEHUB_LOG_LEVEL_INF=y